themes/hugo-theme-altinn/layouts/partials/header.html

```diff
@@ -104,6 +104,6 @@ <h2>{{T "on-this-page"}}</h2>
     {{- $value := (printf "<a href='%s'>%s</a> <span class='adocs-breadcrumb-divider'> / </span> %s" .page.Parent.URL .page.Parent.LinkTitle .value) -}}
     {{- template "breadcrumb" dict "page" .page.Parent "value" $value -}}
   {{- else -}}
-    {{- .value|safeHTML -}}
```

Comment:
We're trying to simplify and shorten the breadcrumbs, and also add language suppert. Drop this change in this PR.